roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -55,6 +55,7 @@
     - "{{ docker_registry_service_ip.results.clusterip }}"
     - "{{ openshift_hosted_registry_name }}.default.svc"
     - "{{ openshift_hosted_registry_name }}.default.svc.{{ openshift.common.dns_domain }}"
+    - "{{ openshift_hosted_registry_routehost | default(omit) }}"
```

Comment:
`omit` fails in the negative case since we expect each item in this list to be a string. My proposed fix is to filter non-strings from the list, PR here https://github.com/openshift/openshift-ansible/pull/4197